bundles/binding/org.openhab.binding.mios/src/main/java/org/openhab/binding/mios/internal/config/DeviceBindingConfig.java

```diff
@@ -193,6 +193,12 @@
 				"urn:upnp-org:serviceId:Dimming1/StepUp()");
 		COMMAND_DEFAULTS.put("DECREASE",
 				"urn:upnp-org:serviceId:Dimming1/StepDown()");
+		COMMAND_DEFAULTS.put("UP",
+				"urn:upnp-org:serviceId:WindowCovering1/UP()");
```

Comment:
MiOS Action-names are case-sensitive, like their UPnP Counterparts, so these defaults would need to be `Up()`, `Down()` and `Stop()` respectively.